src/topology/vector_bundle.lean

```diff
@@ -277,7 +276,21 @@ structure topological_vector_bundle_core (ι : Type*) :=
 (coord_change_comp : ∀ i j k, ∀ x ∈ (base_set i) ∩ (base_set j) ∩ (base_set k), ∀ v,
   (coord_change j k x) (coord_change i j x v) = coord_change i k x v)
 
-attribute [simp, mfld_simps] topological_vector_bundle_core.mem_base_set_at
```

Comment:
why did you remove the simp attribute here?

Reply:
Because now it's taken care of by the new simps below and the Linter does not allow you to have a simp for something that simp already takes care of